src/styles/index.css

```diff
@@ -0,0 +1,261 @@
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  display: inline-block;
+  margin-left: 0;
+  height: 60px;
+  line-height: 60px;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  transition: background-color 0.3s;
+  border-radius: 10px;
+  border-style: 10px solid;
+}
+
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px); /* 4 colunas */
+  gap: 46px 48px;
+  padding: 50px 40px; /* Espaço ao redor */
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .card-container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  outline: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  transition: all 300ms ease;
+  transform: scale(1);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-top: 33px;
+  margin-left: 20px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  position: relative;
+  display: flex;
+  align-items: center;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
+  margin-left: 17px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card__code {
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  width: 98px;
+  height: 14px;
+  margin-left: 17px;
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.card__star {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  width: 166px;
+  height: 16px;
+  margin-left: 17px;
+  margin-bottom: 24px;
+  color: #060b35;
+}
+
+.card__price {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: 18px;
+  width: 166px;
+  height: 18px;
+  margin-left: 17px;
+  margin-bottom: 16px;
+}
+
+.card__price-1 {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-2 {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__bottom {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #00acdc;
+  width: 166px;
+  height: 40px;
+  border-style: 1px Solid #00acdc;
+  border-radius: 5px;
+  margin-left: 17px;
+  transition: all 300ms ease;
+}
+
+.card__bottom-link {
+  font-size: 14px;
+  color: white;
+  text-decoration: none;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.card:hover {
+  transform: scale(1.2);
+}
+
+.card__bottom:hover {
+  background: #fff;
+  border-style: 1px Solid #00acdc;
```

Comment:
The `border-style` property is incorrectly used here. It should be `border` or `border-width` to define the border's width, style, and color. Consider changing it to `border: 1px solid #00acdc;`.